kolena/_experimental/quality_standard.py

```diff
@@ -11,26 +11,57 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import enum
 import json
+from collections import defaultdict
 from dataclasses import asdict
 from typing import Any
 from typing import Dict
 from typing import List
+from typing import Literal
+from typing import Optional
 from typing import Tuple
 from typing import Union
 
+import numpy as np
 import pandas as pd
 
 from kolena._api.v1.event import EventAPI
+from kolena._api.v2._api import GeneralFieldFilter
+from kolena._api.v2._filter import Filters
+from kolena._api.v2._filter import ModelFilter
+from kolena._api.v2._metric import Metric
+from kolena._api.v2._testing import Path as TestingPath
+from kolena._api.v2._testing import StratificationType
+from kolena._api.v2._testing import StratifyFieldSpec
+from kolena._api.v2._testing import TestingRequest
+from kolena._api.v2._testing import TestingResponse
+from kolena._api.v2.dataset import EntityData
+from kolena._api.v2.model import ModelWithEvalConfig
 from kolena._api.v2.quality_standard import CopyQualityStandardRequest
 from kolena._api.v2.quality_standard import Path
+from kolena._api.v2.quality_standard import QualityStandardResponse
+from kolena._experimental.utils import get_delta_percentage
+from kolena._experimental.utils import get_quantile
+from kolena._experimental.utils import margin_of_error
+from kolena._experimental.utils import ordinal
 from kolena._utils import krequests_v2 as krequests
 from kolena._utils import log
 from kolena._utils.instrumentation import with_event
+from kolena._utils.serde import from_dict
 from kolena.dataset.dataset import _load_dataset_metadata
+from kolena.dataset.evaluation import _get_eval_config_id
+from kolena.dataset.evaluation import _get_model_id
 from kolena.errors import IncorrectUsageError
 
 
+class PerformanceDelta(str, enum.Enum):
```

Comment:
Do these serialize well to CSV? Will users ever be expected to know about `PerformanceDelta`? In general, for anything that a user deals with, I'd prefer to use something like
```
PerformanceDelta = Literal["improved", "regressed", "similar", "unknown"]
```

So that the user doesn't need to know about some random enum type, and can instead just work with things as strings.

Reply:
I can confirm export to CSV works fine, but will still change to Literal so the user don't have to see the enum